deploy/assisted-installer-controller/assisted-installer-controller-pod.yaml.template

```diff
@@ -98,6 +110,11 @@ spec:
         - name: host-resolv-conf
           hostPath:
             path: /etc/resolv.conf
+        {{if .SNO}}
+        - name: sno-bootstrap-kubeconfig
+          hostPath:
+            path: /etc/kubernetes/bootstrap-secrets/kubeconfig
```

Comment:
mount the directory instead.
It should allow the cornet case of starting the controller without the bootstrap kubeconfig (it should fall back to the default kubeconfig).